pytests/tests/test_repo_missing_metadata.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (C) 2021 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the GNU General Public License v2 (the "License");
+# you may not use this file except in compliance with the License. The terms
+# of the License are located in the COPYING file of this distribution.
+#
+#   Author: Oliver Kurth <okurth@vmware.com>
+
+import os
+import shutil
+import pytest
+import errno
+
+REPODIR='/root/repofrompath/yum.repos.d'
```

Comment:
Use `REPODIR=/tmp/...` if test fails, our root directory won't contain these files

Reply:
Should be okay since we clean it up, but I changed it anyway.